### PR TITLE
Instructions and code for running locally with current version

### PR DESCRIPTION
**Pull request by dylnuge**

### CODE REVIEW

File: wsgi.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from mcgj import create_app
```
Comment (SaraBee) [inline wsgi.py:1]: Did you mean to check this file in?

**Comment by nebkor** [comment]
I also just got this working locally, and was working on a doc for what to do :) If you create a `.env` file in the top of the repo with the following entries:

```sh
SPOTIPY_REDIRECT_URI=http://127.0.0.1:5000/
SPOTIPY_CLIENT_ID=REDACTED
SPOTIPY_CLIENT_SECRET=REDACTED
# oauth vars
CLIENT_ID=REDACTED
CLIENT_SECRET=REDACTED
CLIENT_CALLBACK=http://127.0.0.1:5000/auth/callback
# enable hot reload of the flask app and in-browser debugger
FLASK_ENV=development
FLASK_APP=mcgj
```

Then you can just run `python3 -m flask run` in that same dir (after setting up the oauth app on recurse and installing the deps).

File: wsgi.py (hunk 1 above)
Comment (dylnuge) [reply wsgi.py:1]: Yeah, this allows local usage of `flask run`. Is there a different way to do it?

**Comment by dylnuge** [comment]
@nebkor Yep! I can add a `.env.sample` or similar to this to capture those 😄

File: wsgi.py (hunk 1 above)
Comment (nebkor) [reply wsgi.py:1]: Yep! Just set the `FLASK_APP` to `mcgj` in your environment (either in the .env file or in your shell), then in the root of the repo, run `python3 -m flask run`.

**Comment by SaraBee** [comment]
I completely forgot about this PR! I think @nebkor addressed this separately, closing this one out.

**Pull request closed without merging**